vizro-core/examples/scratch_dev/app.py

```diff
@@ -1,49 +1,60 @@
-import vizro.plotly.express as px
-from vizro import Vizro
-from vizro._themes._color_values import COLORS
+from dash import Dash, html
+from vizro.figures.library import kpi_card, kpi_card_reference
 
-pastry = pd.DataFrame(
-    {
-        "pastry": [
-            "Scones",
-            "Bagels",
-            "Muffins",
-            "Cakes",
-            "Donuts",
-            "Cookies",
-            "Croissants",
-            "Eclairs",
-            "Brownies",
-            "Tarts",
-            "Macarons",
-            "Pies",
-        ],
-        "Profit Ratio": [-0.10, -0.15, -0.05, 0.10, 0.05, 0.20, 0.15, -0.08, 0.08, -0.12, 0.02, -0.07],
-    }
-)
+df_kpi = pd.DataFrame({"Actual": [100, 200, 700], "Reference": [100, 300, 500], "Category": ["A", "B", "C"]})
+
+# Add single CSS file figures.css or
+base = "https://cdn.jsdelivr.net/gh/mckinsey/vizro@tidy/remove-vizro-classnames/vizro-core/src/vizro/static/css/"
+vizro_bootstrap = base + "vizro-bootstrap.min.css"
 
+# Add entire assets folder from Vizro
+app = Dash(external_stylesheets=[vizro_bootstrap])
 
-page = vm.Page(
-    title="Diverging bar",
-    components=[
-        vm.Graph(
-            figure=px.bar(
-                pastry.sort_values("Profit Ratio"),
-                orientation="h",
-                x="Profit Ratio",
-                y="pastry",
-                color="Profit Ratio",
-                color_continuous_scale=COLORS["DIVERGING_RED_CYAN"],
-            ),
+app.layout = html.Div(
+    [
+        html.H1(children="Title of Dash App"),
+        html.Div(
+            children=[
+                dbc.Row(
+                    [
+                        dbc.Col(
+                            kpi_card(
+                                data_frame=df_kpi,
+                                value_column="Actual",
+                                value_format="${value:.2f}",
+                                icon="shopping_cart",
+                                title="KPI Card I",
+                            )
+                        ),
+                        dbc.Col(
+                            kpi_card_reference(
+                                data_frame=df_kpi,
+                                value_column="Actual",
+                                reference_column="Reference",
+                                icon="payment",
+                                title="KPI Card II",
+                            )
+                        ),
+                        dbc.Col(
+                            kpi_card_reference(
+                                data_frame=df_kpi,
+                                value_column="Reference",
+                                reference_column="Actual",
+                                icon="payment",
+                                title="KPI Card III",
+                            )
+                        ),
+                    ]
+                ),
+            ],
         ),
     ],
+    # **{"data-bs-theme": "light"}
```

Comment:
@antonymilne - I just tested these out. 

**1. Use of vizro bootstrap - default:** If you don't specify any `data-bs-theme`, the default will be dark as discussed. It looks a bit off because it's on a white background, but that's expected as we didn't provide any `bg-color` to the general body container in our `vizro-bootstrap` yet.

![Screenshot 2024-10-15 at 09 16 39](https://github.com/user-attachments/assets/88f7f1c1-e219-4654-a56d-1324ae33192c)

**2. Use of vizro bootstrap - light theme** This is how it looks like if you uncomment this line and add `**{"data-bs-theme": "light"}`:
![Screenshot 2024-10-15 at 09 15 37](https://github.com/user-attachments/assets/e387e266-2bf7-441a-96d6-f77f312fe639)

**3. Usage of any other bootstrap theme:** I just checked the dbc.themes.BOOTSTRAP but it looks as expected and default to their default theme.

Reply:
I think that's perfect, and nothing more to test. Thank you!

If it turns out in the future that adding `bg-color` to match the dark default is annoying or awkward for some reason then I'm happy to turn our default back to the light theme btw. My preference for default = dark is just a weak one.